elements/pl-number-input/pl-number-input.mustache

```diff
@@ -15,16 +15,25 @@
             <span class="input-group-text">{{label}}</span>
         </span>
         {{/label}}
-        <input name="{{name}}" type="text" class="form-control" size="35" {{^editable}}disabled{{/editable}} {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}} aria-describedby="basic-addon1" placeholder="{{shortinfo}}" />
+        <input name="{{name}}" type="text" class="form-control {{^display_append_span}}pl-number-input-force-radius-border{{/display_append_span}}" size="{{size}}" {{^editable}}disabled{{/editable}} {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}} aria-describedby="basic-addon1" placeholder="{{shortinfo}}" />
+        {{#display_append_span}}
         <span class="input-group-append">
-            {{#suffix}}<span class="input-group-text">{{suffix}}</span>{{/suffix}}
+            {{#suffix}}<span class="input-group-text {{^shortinfo}}pl-number-input-force-radius-border{{/shortinfo}}">{{suffix}}</span>{{/suffix}}
+            {{#shortinfo}}
             <a class="btn btn-light border" role="button" data-toggle="popover" data-html="true" title="Number" data-content="{{info}}" data-placement="auto" data-trigger="focus" tabindex="0">
                 <i class="fa fa-question-circle" aria-hidden="true"></i>
                 {{#correct}}&nbsp;<span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>{{/correct}}
                 {{#partial}}&nbsp;<span class="badge badge-warning"><i class="fa fa-circle-o" aria-hidden="true"></i> {{partial}}%</span>{{/partial}}
                 {{#incorrect}}&nbsp;<span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i> 0%</span>{{/incorrect}}
             </a>
+            {{/shortinfo}}
         </span>
+        {{/display_append_span}}
+        {{^shortinfo}}
+            {{#correct}}&nbsp;<span class="badge badge-success pl-number-input-inline-badge"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>{{/correct}}
```

Comment:
These lines duplicate what’s above. It would be nice to have them appear only once. I think you can change the template a bit to make this work?